Adds two missing parameters for SignTool

tstewart65 opened the pull request:
\csp - Cryptographic Service Provider
\kc - Private Key Container Name

Fix for #4404

augustoproiete reviewed (approved):
LGTM

devlead reviewed (changes requested):
LGTM, could you please add som tests in
https://github.com/cake-build/cake/blob/124704e625c670d7469fd9fba1b86596f2883a5c/src/Cake.Common.Tests/Unit/Tools/SignTool/SignToolSignRunnerTests.cs#L15

tstewart65 commented:
I have added the tests for the two new parameters.  I also moved the two new parameters to just before the target assemblies.  That way they are at the end of all the parameters that were already there.

devlead commented:
@tstewart65 your changes have been merged, thanks for your contribution 👍
